common-utils/target/surefire-reports/TEST-org.common.utils.LedgerInfoTest.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
How this xml is written , is it generated or written we can speak on this

Reply:
i executed mvn command i think during that time this might generated